### ISSUE TITLE
表記揺れいくつか

### ISSUE THREAD

**Issue by jidaikobo-shibata** [waic/wcag21#1876]
2.2に持っていくときに、役に立たないかしらと、WCAG 2.1（本体・解説書・達成方法集）について、表記が揺れている言葉をgrepとwcで数えてみました。以下の通りまとめてみましたが、もし検討いただけるものがあるようでしたら幸いです。

又は: 2328項目
または: 138項目

及び: 2114項目
および: 61項目

アクセシビリティ サポーテッド: 170
アクセシビリティ・サポーテッド: 65

ユーザエージェント: 656
ユーザーエージェント: 4
これについては[4.1.1の注記内](https://dev.waic.jp/translations/WCAG21/#parsing)でも揺れていて、近所なので気になります。

入れ子: 33
ネスト: 12
これは文脈次第で読みやすさが違うのかもしれません。ので指摘すべきでもないのかもしれませんが、でも、やっぱりincorrect element nestingを「ネスト」とし、inappropriately nested elementsを「入れ子」にしているのは、なぜかしら、という気になりました。

制作者: 409
製作者: 7
「製作者（Author）」は、[ARIA24](https://dev.waic.jp/translations/WCAG21/Techniques/aria/ARIA24.html)に集中（[3.1.5](https://dev.waic.jp/translations/WCAG21/Understanding/reading-level.html)、[G60](https://dev.waic.jp/translations/WCAG21/Techniques/general/G60.html)にもある）。

スクリーンリーダー: 236
スクリーン・リーダー: 2
「スクリーン・リーダー」は、[F96](https://dev.waic.jp/translations/WCAG21/Techniques/failures/F96.html)と[G211](https://dev.waic.jp/translations/WCAG21/Techniques/general/G211.html)

ブラウザ: 244
ブラウザー: 2
「ブラウザー」は[F96](https://dev.waic.jp/translations/WCAG21/Techniques/failures/F96.html)と[G149](https://dev.waic.jp/translations/WCAG21/Techniques/general/G149.html)に。

インタフェース: 284
インターフェース: 29
インターフェイス: 4

**Comment by momdo** [comment]
ありがとうございます。
たくさんいただいているので、わけてコメントを返せていければと。

> 又は: 2328項目
または: 138項目
>
> 及び: 2114項目
および: 61項目

ご指摘の「および」「または」がなぜかWCAG 2.1訳に紛れ込んでますね…これは修正したいと思います。

https://dev.waic.jp/translations/WCAG21/#comparison-with-wcag-2-0

> WCAG 2.1 は、認知または学習障害のある利用者、ロービジョンの利用者、モバイルデバイス上の障害のある利用者、という三つの主要なグループためのアクセシビリティのガイダンスを改善することを目的に開始された。

https://www.w3.org/TR/2023/REC-WCAG21-20230921/#comparison-with-wcag-2-0

> WCAG 2.1 was initiated with the goal to improve accessibility guidance for three major groups: users with cognitive or learning disabilities, users with low vision, and users with disabilities on mobile devices.

「グループための」がそもそもおかしいですが、andの訳どこいった…

---

https://dev.waic.jp/translations/WCAG21/#pointer-cancellation

キーボードまたはテンキーパッドのキープレスをエミュレートする機能は必須と考えられる。

https://www.w3.org/TR/2023/REC-WCAG21-20230921/#pointer-cancellation

Functions that emulate a keyboard or numeric keypad key press are considered essential.

「又は」にすればよいと思いますが、essentialは必要不可欠と訳さないとダメですね…😇

---

https://dev.waic.jp/translations/WCAG21/#wcag-2-1-supporting-documents

> これには WCAG 2.0 および WCAG 2.1 のコンテンツが含まれており、関連するコンテンツにコンテンツ制作者がフォーカスできるよう、様々な方法でフィルタリングできるようになっている。

https://www.w3.org/TR/2023/REC-WCAG21-20230921/#wcag-2-1-supporting-documents

> This includes content from WCAG 2.0 and WCAG 2.1 and can be filtered in many ways to help authors focus on relevant content.

これは単に「及び」に置換するとよさそうです。

---

https://dev.waic.jp/translations/WCAG21/#parsing

> この基準が作成されて以降、HTML Living Standard は、ユーザーエージェントが不完全なタグ、不正な要素のネスト、重複した属性、および一意でない ID をどのように処理しなければならないかを規定する明確な要件を採用した。[HTML]

https://www.w3.org/TR/2023/REC-WCAG21-20230921/#parsing

> Since this criterion was written, the HTML Living Standard has adopted specific requirements governing how user agents must handle incomplete tags, incorrect element nesting, duplicate attributes, and non-unique IDs. [HTML]

こちらも「及び」に置換すればよいですね。ここにはご指摘の「ネスト」「ユーザーエージェント」も紛れ込んでいますが、詳細は続きのコメントで。

---

「および」「または」を単に置換すればよいのではという説もありますが、上記のように訳が不正確な箇所も含まれることがあるので、解説書・達成方法集については、出くわしたら見返す形にしていければと思います（ということで解説書・達成方法集は今回はご容赦いただければと）。

**Comment by momdo** [comment]
> アクセシビリティ サポーテッド: 170
アクセシビリティ・サポーテッド: 65

これはかつてのAS情報が「アクセシビリティ・サポーテッド」と記載されていたことに起因していると思います。
そのため、AS情報へのリンクテキストがそのようになっている、ということですね。
例： https://waic.jp/translations/WCAG21/Techniques/html/H91#example-1-links

…ところで、[アクセシビリティ・サポーテッド（AS）情報: H91-1c](https://waic.jp/translations/as/info/201406/H91-1c.html)のリンクがディレクトリ変更に伴って、リンクが切れてますね…。

ここでURLと一緒にリンクテキストも直したほうがよいのか、Techniques for WCAG 2.2での対応としたほうがよいのかはご相談と言ったところでしょうか…。

**Comment by momdo** [comment]
> ユーザエージェント: 656
ユーザーエージェント: 4

これに関しては純粋に揺れてしまっているだけなので、置換したいと思います。

**Comment by momdo** [comment]
> 入れ子: 33
ネスト: 12
これは文脈次第で読みやすさが違うのかもしれません。ので指摘すべきでもないのかもしれませんが、でも、やっぱりincorrect element nestingを「ネスト」とし、inappropriately nested elementsを「入れ子」にしているのは、なぜかしら、という気になりました。

どこを比較してそう言及されているのかがわかりませんが、いずれにせよWCAG 2.1の中では「ネスト」が1箇所だけで、ほかは「入れ子」となっているので、「入れ子」に統一するのが適当そうです。

**Comment by momdo** [comment]
> 制作者: 409
製作者: 7
「製作者（Author）」は、ARIA24に集中（3.1.5、G60にもある）。

ARIA24に集中しているのは、最初に訳を起こした私が犯人ですね…。ついでに訳を見直すのが適当かもしれませんが、WCAG 2.1訳には含まれていないので、修正のタイミングはご相談かなと。

**Comment by momdo** [comment]
> スクリーンリーダー: 236
スクリーン・リーダー: 2
「スクリーン・リーダー」は、F96とG211
>
> ブラウザ: 244
ブラウザー: 2
「ブラウザー」はF96とG149に。

どちらも達成方法集ですので、ここでしれっと置換すればよいですかね…

**Comment by momdo** [comment]
> インタフェース: 284
インターフェース: 29
インターフェイス: 4

> 7. ユーザインタフェース コンポーネントの入力目的

> 達成基準 3.2.1 フォーカス時
> いずれのユーザインターフェイス コンポーネントも、

> 達成基準 3.2.2 入力時
> ユーザインタフェース コンポーネントの設定を変更することが、

これはひどい…いまのところは「インタフェース」が正だと思いますので、ひとまずそのようにあわせたいと思います。
それはそれとして“長音を省かないといけない”というJISの表記ルールはなくなったと思いますので、今後、別の表記（たとえばインターフェース）にあわせるのはありかとは思います。

**Comment by momdo** [comment]
まとめるとこんな感じですかね

@bakera ひとまず「このターン」「後のターン」でより分けてみましたが、解説書・達成方法集にある「および」「または」以外を全部直すという選択肢もあるかと思います。方針を確認いただければと。

- または、および
  - WCAG 2.1訳で揺れているものは直す、解説書・達成方法集はこのターンではママ
- アクセシビリティ・サポーテッド
  - WCAG 2.1訳にはない。おそらく達成方法集
  - かつて「アクセシビリティ・サポーテッド（AS）情報」だったことに起因か
  - このターンではママとしたい
- ユーザーエージェント: 4
  - 4.1.1の注記にあるので、このターンで一括で置換
- 入れ子/ネスト
  - 4.1.1の注記に見えるので、WCAG 2.1訳はひとまず「入れ子」に寄せる
  - ほかは後で置換？
- 製作者、スクリーン・リーダー
  - WCAG 2.1訳には含まれず、あとで一括置換か
- ブラウザー
  - 達成方法集に2箇所なのでこのターンではママ？　それはそれとして長音ルールの見直しのしたさ。
- インタフェース/インターフェース/インターフェイス
  - なんとWCAG 2.1訳で全部が見える…のでこのターンで一括置換する

**Comment by momdo** [comment]
#1893 では解説書内の「製作者」「ネスト」を修正。

**Comment by bakera** [comment]
現状のWCAG 2.2本体に関してはどれも対応済みのように見受けられました。
翻訳用語集に反映しておきました。
このissueはcloseしておきます。
